From c385d001a1d882c8860f3a0c81cdcabee456f2d3 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 20 Jul 2020 16:06:45 +0800
Subject: [PATCH 1/1] Test case for library .h file

Description: Change the define value of 'MAX_KEY_SECTION_LEN' in the MdeModulePkg/Library/DeviceManagerUiLib/DeviceManager.h

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Library/DeviceManagerUiLib/DeviceManager.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Library/DeviceManagerUiLib/DeviceManager.h b/MdeModulePkg/Library/DeviceManagerUiLib/DeviceManager.h
index c53c2a1a0e1a..309c22bf8d8f 100644
--- a/MdeModulePkg/Library/DeviceManagerUiLib/DeviceManager.h
+++ b/MdeModulePkg/Library/DeviceManagerUiLib/DeviceManager.h
@@ -43,7 +43,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define DEVICE_KEY_OFFSET                    0x4000
 #define NETWORK_DEVICE_LIST_KEY_OFFSET       0x2000
 
-#define MAX_KEY_SECTION_LEN                  0x1000
+#define MAX_KEY_SECTION_LEN                  0x1010
 
 #define QUESTION_NETWORK_DEVICE_ID           0x3FFF
 //
-- 
2.27.0.windows.1

